package-res/resources/web/pentaho/type/_type.js

```diff
@@ -414,6 +418,27 @@ define([
 
       _sourceId: null,
 
+      /**
+       * Gets the source module identifier of this type.
+       *
+       * The source identifier is the module id of the _actual_ AMD/RequireJS module
```

Comment:
The source identifier is the module identifier of the _actual_ AMD/RequireJS module